.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for Trino Python client
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Describe the feature
+A clear and concise description of what you want to happen.
```

Comment:
How about asking for a problem statement first, and then the feature description that should explain how it addresses the problem?